x-pack/plugins/security_solution/server/endpoint/routes/policy/service.ts

```diff
@@ -12,8 +12,17 @@ export function getESQueryPolicyResponseByHostID(hostID: string, index: string)
   return {
     body: {
       query: {
-        match: {
-          'host.id': hostID,
+        bool: {
+          filter: {
+            term: {
+              'host.id': hostID,
+            },
+          },
+          must_not: {
+            term: {
+              'Endpoint.policy.applied.name': 'initial',
```

Comment:
could we filter by the id?  I believe the initial policy will have an all `000...` UUID.  I'm just afraid of users creating a policy called `initial`.  It could happen

Reply:
:100: Still works, and then user won't collide with the filter